config/config.go

```diff
@@ -32,8 +33,11 @@ type Config struct {
 	HaProxyPidFile               string           `yaml:"haproxy_pid_file"`
 	IsolationSegments            []string         `yaml:"isolation_segments"`
 	ReservedSystemComponentPorts []int            `yaml:"reserved_system_component_ports"`
+	DrainWait                    time.Duration    `yaml:"drain_wait"`
```

Comment:
**Suggestion**

Since it's a duration, I think being a little verbose and naming the property as `DrainWaitDuration` and yaml as `drain_wait_duration` is more descriptive. Without context, it could be interpreted as a boolean. That said, I'm okay with it either way.

Reply:
I agree with Andy here too.